src/main/java/com/sap/oss/phosphor/fosstars/data/artifact/experimental/VulnerabilitiesInArtifact.java

```diff
@@ -0,0 +1,61 @@
+package com.sap.oss.phosphor.fosstars.data.artifact.experimental;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.feature.AbstractFeature;
+import com.sap.oss.phosphor.fosstars.model.value.UnknownValue;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerabilities;
+import com.sap.oss.phosphor.fosstars.model.value.VulnerabilitiesValue;
+import com.sap.oss.phosphor.fosstars.util.Json;
+import java.io.IOException;
+
+public class VulnerabilitiesInArtifact extends AbstractFeature<Vulnerabilities> {
```

Comment:
This class duplicates `VulnerabilitiesInProject`. Let's please try to avoid duplicate code. Please use `VulnerabilitiesInProject`. Then, we'll rename it to `VulnerabilitiesFeature`.